control/statefbk.py

```diff
@@ -422,22 +474,24 @@ def obsv(A, C):
     n = np.shape(amat)[0]
 
     # Construct the controllability matrix
-    obsv = cmat
+    O = cmat
     for i in range(1, n):
-        obsv = np.vstack((obsv, cmat*amat**i))
-    return obsv
+        O = np.vstack((O, cmat * amat ** i))
+    return O
+
 
-def gram(sys,type):
+def gram(sys, desired_computation_str):
```

Comment:
This seems very cumbersome as can named parameter.  Was there a reason to switch from `type`?  If so, can we find something short (ideally one word)?

Reply:
I don't like the idea of a variable named as a build-in function: 
https://docs.python.org/3/library/functions.html#type

I suggest `gramian_type`, what do you think about it?